app/helpers/cases_helper.rb

```diff
@@ -116,6 +116,12 @@ def action_button_for(event)
               case_sar_extensions_path(@case),
               id: 'action--remove-extended-deadline-for-sar',
               class: 'button-secondary', method: :delete
+    when :record_data_request
+      btn_type = @case.current_state == 'data_to_be_requested' ? 'secondary' : 'tertiary'
+      link_to 'Record data request',
+              new_case_data_request_path(@case),
+              id: 'action--record-data-request',
+              class: "button-#{btn_type}", method: :get
```

Comment:
I thought we were getting rid of this `method: :get`? :) 

not a problem though, we can change it later. Only worry is, is this back in because of a merge issue and is there anything else that could have gotten mixed up too? Or was it not part of the PRs

Reply:
Hah forgot to remove it post-merge!